install/README.md

```diff
@@ -54,10 +54,16 @@ helm install --name wavefront wavefront/wavefront-operator --set wavefront.url=h
 
 ## Uninstallation
 To uninstall/delete a release named "wavefront":
+helm2:
 ```
 helm delete wavefront --purge
 ```
 
+helm3:
+```
+helm delete wavefront
```

Comment:
helm 3 requires the namespace to be provided to the delete command via the `-n <namespace>` flag